crates/proof-of-sql/src/proof_primitive/dory/dynamic_dory_commitment_helper_cpu.rs

```diff
@@ -61,6 +61,11 @@ fn compute_dory_commitment(
         CommittableColumn::RangeCheckWord(column) => {
             compute_dory_commitment_impl(column, offset, setup)
         }
+        CommittableColumn::FixedSizeBinary(_byte_width, column) => {
+            // FIXME: Is this interpretation correct? or should we chunk the bytes
+            // into `byte_width`-sized chunks and convert each chunk into a scalar?
+            compute_dory_commitment_impl(column, offset, setup)
```

Comment:
I believe something like this may work:
```suggestion
            // FIXME: Is this interpretation correct? or should we chunk the bytes
            // into `byte_width`-sized chunks and convert each chunk into a scalar?
            compute_dory_commitment_impl(&column.iter().chunks_exact(byte_width).collect_vec(), offset, setup)
```

Reply:
Be sure to add some tests here ensuring that this is behaving correctly. Even though CI is currently passing, this code is currently incorrect.

Reply:
Alright, I’ve updated that based on your suggestion and also added a test `we_can_compute_a_dory_commitment_with_fixed_size_binary_values` that is currently failing with the following error:
```sh
panicked at crates/proof-of-sql/src/proof_primitive/dory/pack_scalars.rs:179:14:

source slice length (1) does not match destination slice length (4)
```

tracing it back, I found that the `pack_bit` function is the one panicking. It seems to be related to the implementation of `OffsetToBytes`.
```rust
column.iter().enumerate().for_each(|(i, value)| {
	let index = i + offset;
	let row_offset = (index % num_columns) * bit_table_sum_in_bytes;
	let col_offset = current_byte_size * (index / num_columns);
	let offset_index = row_offset + col_offset + byte_offset;

	packed_scalars[offset_index..offset_index + current_byte_size]
		.copy_from_slice(&value.offset_to_bytes()[..]);
});
```
The issue is that it’s currently iterating over values and calling `offset_to_bytes`, treating each element in the column as its own value. 

This is not ideal for `FixedSizeBinary` because multiple consecutive values form a single value (`byte_size`). I’m not quite sure how to handle this properly. Could you point me in the right direction?

Reply:
Any updates on this? I’ve rebased on top of main @JayWhite2357

Reply:
This is still marked as draft. Is it ready for review?

Reply:
Marked it as ready for review. The test `we_can_compute_a_dory_commitment_with_fixed_size_binary_values` is still failing the CI, though. Let me know what you think